Tests/app/config/injection_test.yml

```diff
@@ -1,7 +1,6 @@
 parameters:
-    ongr_project.sql_relations.simple_trigger.class: ONGR\ConnectionsBundle\Sync\Extractor\Relation\SimpleSqlRelation
-    ongr_project.sql_relations.composed_trigger.class: ONGR\ConnectionsBundle\Sync\Extractor\Relation\ComposedSqlRelation
-    ongr_project.sql_relations.related_table.class: ONGR\ConnectionsBundle\Sync\Extractor\Relation\JoinStatement
+    ongr_project.extractor.descriptor.class: ONGR\ConnectionsBundle\Sync\Extractor\Descriptor\ExtractionDescriptor
```

Comment:
this should be defined in here, use default parameters from connections, like `ongr_connections.extractor.descriptor.class`